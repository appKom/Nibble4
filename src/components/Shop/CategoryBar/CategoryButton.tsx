import React, { FC } from "react";
import styled from "styled-components";
import { OnlineBlue, OnlineOrange } from "style";

type CategoryButtonProps = {
  text: string;
};

const CategoryButton: FC<CategoryButtonProps> = ({
  text,
}: CategoryButtonProps) => {
  return <Button> {text} </Button>;
};

export default CategoryButton;

const Button = styled.button`
  background-color: ${OnlineBlue};
  color: white;
  border: none;
  border-radius: 2px;
  margin: 4px;
  width: 6%;
  height: 25px;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);

  :active {
    background-color: ${OnlineOrange};
  }

  :focus {
    outline: none;
  }
`;
