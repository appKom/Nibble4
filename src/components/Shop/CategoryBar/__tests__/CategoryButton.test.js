import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import CategoryButton from "../CategoryButton";

afterEach(cleanup);
it("renders", () => {
  const { asFragment } = render(<CategoryButton category="Alt" />);
  expect(asFragment()).toMatchSnapshot();
});
