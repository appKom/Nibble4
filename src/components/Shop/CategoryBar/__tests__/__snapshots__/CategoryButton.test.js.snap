// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <button
    class="sc-bdfBwQ dRtQbr"
    style="background-color: rgb(249, 183, 89);"
  >
    Alt
  </button>
</DocumentFragment>
`;
